docker-compose.yaml

```diff
@@ -55,6 +55,35 @@ services:
         build:
             context: api
             dockerfile: Dockerfile
+    poseidon_certstrap:
```

Comment:
As these are specific to Faucet, this should go into this docker-compose file instead: https://github.com/CyberReboot/poseidon/blob/master/helpers/faucet/docker-compose.yaml

Reply:
Moved - thanks!